### ISSUE TITLE
Configration file not saving

### ISSUE THREAD

**Issue by bhaumik-tripathi** [vikdevelop/SaveDesktop#186]
### Your question

On clicking "Save" and selecting the location, the application does show that the configuration have been saved and directs to the selected folder through the popup, but the configuration file does not come in the folder. I am using ubuntu 22.04 and installed the application from [flathub](https://flathub.org/apps/io.github.vikdevelop.SaveDesktop). Please ask for any other required information. I installed the application twice but no changes occurred and the problem persisted.

**Comment by vikdevelop** [comment]
Hello, please run the SaveDesktop app in terminal (Ctrl+Alt+T) and type `flatpak run io.github.vikdevelop.SaveDesktop` and copy the output and paste it here.

**Comment by bhaumik-tripathi** [comment]
On trying and waiting for few mins, it opens the  application with following output on the terminal
```
/app/main_window.py:86: DeprecationWarning: Adw.ViewSwitcherTitle.set_stack is deprecated
  self.switcher_title.set_stack(self.stack)
/app/main_window.py:87: DeprecationWarning: Adw.ViewSwitcherTitle.set_title is deprecated
  self.switcher_title.set_title("")

(main_window.py:3): Adwaita-CRITICAL **: 14:40:01.630: adw_status_page_set_child: assertion 'gtk_widget_get_parent (child) == NULL' failed

(main_window.py:3): Adwaita-CRITICAL **: 14:40:01.631: adw_status_page_set_child: assertion 'gtk_widget_get_parent (child) == NULL' failed
cp: cannot stat '/home/sandesh/.local/share/gnome-background-properties': No such file or directory
cp: cannot stat '/home/sandesh/.local/share/nautilus-python': No such file or directory
cp: cannot stat '/home/sandesh/.local/share/gnome-control-center': No such file or directory
/bin/sh: -c: line 1: syntax error near unexpected token `('
/bin/sh: -c: line 1: `tar --gzip -cf gnome_customization_(part_1).sd.tar.gz ./'
mv: cannot stat '*.tar.gz': No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
```

**Comment by bhaumik-tripathi** [comment]
I tried it once before also, at that time the command did nothing and the cursor kept on blinking after clicking enter. On terminating the action by control c , it showed following
```
Traceback (most recent call last):
  File "/app/main_window.py", line 9, in <module>
    from open_wiki import *
  File "/app/open_wiki.py", line 10, in <module>
    if "The requested URL returned error: 404" in subprocess.getoutput(f"curl --head --fail https://raw.githubusercontent.com/vikdevelop/SaveDesktop/main/translations/wiki/{r_lang}.xml"):
                                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 691, in getoutput
    return getstatusoutput(cmd, encoding=encoding, errors=errors)[1]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 671, in getstatusoutput
    data = check_output(cmd, shell=True, text=True, stderr=STDOUT,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 550, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 1196, in communicate
    stdout = self.stdout.read()
             ^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
```
In case it may be of any use( though I agree that I didn't waited for long).

**Comment by vikdevelop** [comment]
You can try to set the filename without ( ). For me, the app launches quickly, so I don't know why it's slow to launch, however I'll try to look into it.

**Comment by bhaumik-tripathi** [comment]
thanks @vikdevelop , the file is saved in the desired folder. I think that it was the problem due to those parenthesis but please look into why it takes so much time to open. Thanks again.

**Comment by vikdevelop** [comment]
I released a new version, 2.9.9, and slow loading should be fixed. I tested it on the Cinnamon, KDE and GNOME desktops.
